.chloggen/splunkent-client-refactor-with-sdk.yaml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: splunkentreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Refactor Splunkenterprise Receiver component to better leverage existing otel SDK."
```

Comment:
I would change the note to be user facing saying that you can use the auth settings to connect to Splunk enterprise.